crates/viewer/re_space_view_spatial/src/scene_bounding_boxes.rs

```diff
@@ -42,6 +47,17 @@ impl SceneBoundingBoxes {
                 .data()
                 .and_then(|d| d.downcast_ref::<SpatialViewVisualizerData>())
             {
+                // If we're in a 3D space, but the visualizer is distintivly 2D, don't count it towards the bounding box.
+                // These visualizers show up when we're on a pinhole camera plane which itself is heuristically fed by the
+                // bounding box, creating a feedback loop if we were to add it here.
+                if space_kind == SpatialSpaceViewKind::ThreeD
+                    && data
+                        .preferred_view_kind
+                        .map_or(true, |kind| kind != space_kind)
```

Comment:
If there is no `preferred_view_kind`, then it counts as "distintivly 2D" ?

Reply:
just before leaving for lunch I also stumbled into this as well - so turn out only the camera is "indecisive" about this and opting to "in doubt add to the box" actually makes the problem resource. But as is now the camera doesn't add to the bounding box when it should add its center. Doesn't quite add up though, if it only adds its center why would that cause flickering.
I need to have a closer look!

Reply:
ah! no it still works with `.map_or(false, |kind| kind != space_kind)`, I accidentally tested the wrong version